src/shared/input-wrapper/input-wrapper.tsx

```diff
@@ -92,7 +92,7 @@ export const InputWrapper = styled(InputBox)<InputWrapperStyleProps>`
     position: relative;
     height: max-content;
     width: 100%;
-    padding: 0 ${Spacing["spacing-16"]};
+    padding: 0 0 0 ${(props) => (props.$readOnly ? "0" : Spacing["spacing-16"])};
```

Comment:
hmm I don't think we should remove right padding here. for shorter fields, the text would be right against the edge